jupyterlab_hub/typings/xterm/xterm.d.ts

```diff
@@ -0,0 +1,83 @@
+// Type definitions for term.js 0.0.7
```

Comment:
None of these typings files should be required here (except the url-join).  If they are then we've accidentally leaked a third party API.